--- build/src/openvpn-2.3.2/src/openvpn/openvpn.c.orig	2013-05-31 05:00:07.000000000 -0700
+++ build/src/openvpn-2.3.2/src/openvpn/openvpn.c	2013-11-03 13:31:06.000000000 -0800
@@ -290,37 +290,3 @@
   openvpn_exit (OPENVPN_EXIT_STATUS_GOOD);  /* exit point */
   return 0;			            /* NOTREACHED */
 }
-
-#ifdef WIN32
-int
-wmain (int argc, wchar_t *wargv[]) {
-  char **argv;
-  int ret;
-  int i;
-
-  if ((argv = calloc(argc+1, sizeof(char*))) == NULL)
-    return 1;
-
-  for (i = 0; i < argc; i++)
-    {
-      int n = WideCharToMultiByte (CP_UTF8, 0, wargv[i], -1, NULL, 0, NULL, NULL);
-      argv[i] = malloc (n);
-      WideCharToMultiByte (CP_UTF8, 0, wargv[i], -1, argv[i], n, NULL, NULL);
-    }
-
-  ret = openvpn_main(argc, argv);
-
-  for (i=0; i < argc; i++ )
-    {
-      free (argv[i]);
-    }
-  free(argv);
-
-  return ret;
-}
-#else
-int
-main (int argc, char *argv[]) {
-	return openvpn_main(argc, argv);
-}
-#endif
